docs/core/distribution-packaging.md

```diff
@@ -115,7 +121,7 @@ The following lists the recommended packages:
 - `dotnet-sdk-[major].[minor]` - Installs the latest SDK for specific runtime
   - **Version:** \<sdk version>
   - **Example:** dotnet-sdk-7.0
-  - **Contains:** (3),(4)
+  - **Contains:** (3),(4),(18)
```

Comment:
With the initial implementation that used hard links, the runtime pack directories needed to be part of the runtime distro packages.

With the change to use symbolic links, we can include the directories in the SDK distro package.
That is what they are actually used from.